src/js/components/Tabs/doc.js

```diff
@@ -0,0 +1,42 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils';
+
+import Tab from './Tab';
+
+export default (Tabs) => {
+  const DocumentedTabs = describe(Tabs)
+    .availableAt(getAvailableAtBadge('Tabs'))
+    .description('A tabular view component.')
+    .usage(
+      `import { Tabs, Tab } from 'grommet';
+<Tabs>
+  <Tab title='Tab 1'>...</Tab>
+  <Tab title='Tab 2'>...</Tab>
+</Tabs>`
+    );
+
+  DocumentedTabs.propTypes = {
+    activeIndex: PropTypes.number.description(
+      'Active tab index. If specified, you must call onActive and update activeIndex yourself.'
```

Comment:
I don't quite understand what it means to have the caller call `onActive`, since that is a property of Tabs.

Reply:
`activeIndex` is also a property of Tabs. This is the uncontrolled vs controlled thing. If you provide activeIndex you have to control the changes with the `onActive`. Similar to `value` and `onChange`.

Reply:
I understand the behavior. I find the guidance "you must call onActive" to be confusing, since `onActive` is a property of Tabs. I don't know what it means to "call a property".